tests/utils/test_defence.py

```diff
@@ -251,6 +253,10 @@ def test__gtfs_defence():
         ),
     ):
         _gtfs_defence("tester", "test")
+    # passing test
+    with does_not_raise():
+        gtfs = GtfsInstance(here("tests/data/chester-20230816-small_gtfs.zip"))
+        _gtfs_defence(gtfs=gtfs, param_nm="gtfs")
```

Comment:
This relates to #180 :

"4. Add a unit test for _gtfs_defence() in the passing case (i.e. using a GTFS test fixture)"

* `from contextlib import nullcontext as does_not_raise` is a new one on me. After a bit of googling I found this in the pytest docs. Good to include the link in a comment - https://docs.pytest.org/en/6.2.x/example/parametrize.html#parametrizing-conditional-raising

Reply:
done

Reply:
@CBROWN-ONS have you pushed it up?